changelogs/fragments/46483-role_names-change.yaml

```diff
@@ -0,0 +1,11 @@
+minor_changes:
+  - magic variables - added a new ``ansible_role_names`` magic variable to include the names of roles being applied to
+    the host both directly and indirectly (via dependencies).
+  - magic variabels - added a new ``ansible_play_role_names`` magic variable to mimic the old functionality of
```

Comment:
typo - `variabels` - though as a new word, it sounds pretty . . .

Reply:
Thanks, fixed.